website/cue/reference/remap/functions/find.cue

```diff
@@ -0,0 +1,63 @@
+package metadata
+
+remap: functions: find: {
+	category: "String"
+	description: """
+		Determines whether the elements in the `value` matches the `pattern` and returns its position or -1.
+		"""
+
+	arguments: [
+		{
+			name:        "value"
+			description: "The string."
```

Comment:
```suggestion
			description: "The string to find the substring in."
```

Reply:
Does regex pattern work with "substring"

Reply:
True!

> The string to find the match in.

Could be better then.